tests/foreman/cli/test_remoteexecution.py

```diff
@@ -168,7 +173,25 @@ def test_positive_view_dump(self):
 class RemoteExecutionTestCase(CLITestCase):
     """Implements job execution tests in CLI."""
 
-    @stubbed
+    @classmethod
+    def setUpClass(cls):
+        """Create an organization which can be re-used in tests."""
+        super(RemoteExecutionTestCase, cls).setUpClass()
+        cls.organization = make_org()
+        ssh.command(
+            '''echo 'echo "test"' > {0}'''.format(TEMPLATE_FILE)
```

Comment:
Yes, *hammer job-invocation create --file template* requires a template file on disk, so I pre-create it trough ssh, I didn't find a way to go about it differently